### PR TITLE
[FIX] l10n_ch: fix swiss QR Code crap

### PR DESCRIPTION
**Pull request by qdp-odoo**
The specifications of this feature have not been frozen yet and the feature is not expected before mid-2020. Merging it in stable was premature.
There's now an option to hide the feature until really needed.

Also fixed all 3 typos/grammar errors in the single sentence message displayed when some pieces of information were missing while printing the QR code

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by qdp-odoo** [comment]
@nim-odoo I suppose all these changes are fine for a stable version, what do yo think?

File: addons/l10n_ch/models/res_config_settings.py [hunk 1]
```diff
@@ -18,3 +18,4 @@ class ResConfigSettings(models.TransientModel):
         related="company_id.l10n_ch_isr_scan_line_left", readonly=False)
     l10n_ch_isr_scan_line_top = fields.Float(string='Vertical offset',
         related="company_id.l10n_ch_isr_scan_line_top", readonly=False)
+    l10n_ch_print_qrcode = fields.Boolean("Print Swiss QR Code", config_parameter='l10n_ch.print_qrcode')
```
Comment (nim-odoo) [inline addons/l10n_ch/models/res_config_settings.py]: Nope... Although this is linked to an `ir.config_parameter`, this is still adding a field to `res.config.settings`.

Comment (qdp-odoo) [reply addons/l10n_ch/models/res_config_settings.py]: @rco-odoo fields of res.config.settings that have the attribute config_parameter=True could be computed/not stored, if you ask me....

**Comment by qdp-odoo** [comment]
@nim-odoo I made the requested changes. Can you give me my r+ ? pretty pleeeeaaaase

**Pull request closed without merging**